jsHelper/sidebarConfig.js

```diff
@@ -100,7 +100,7 @@
 
 	function writeStorage() {
 		const array = ordered.map(a => [a[0].dataset.id, a[1]]);
-		console.log(array);
+		// console.log(array); prints path
```

Comment:
You can remove this statement altogether.

Reply:
Lol never knew that existed 😂